docs/usage/mend-hosted/hosted-apps-config.md

```diff
@@ -17,6 +17,36 @@ The Renovate logs for the Mend-hosted apps are on the [Mend Developer Portal](ht
 
 Reading the logs can help you understand the configuration that Renovate used.
 
+## Renovate Version
+
+The Renovate version used by the Mend-hosted apps is updated manually by the maintainers of the app.
+The maintainers don't follow any release schedule or release cadence, but try to update at least once a week.
+This means the Mend Renovate App can lag a few hours to a week behind the open source version.
```

Comment:
This still duplicates the FAQ content too much

Reply:
Would you rather the content be mainly in the FAQs or in the Mend-hosted apps section?

Reply:
Mend-hosted. FAQ is a catchall anyway

Reply:
Removed duplication in FAQs. Replaced FAQ content with a link to the Mend-hosted App Config.